scripts/send_device_status.js

```diff
@@ -9,7 +8,6 @@ Shelly.call("MQTT.GetConfig", {}, function (config) {
 function SendDeviceStatus() {
     let _device_info = Shelly.getDeviceInfo();
     installed_version = _device_info.ver;
-    model = _device_info.model;
     Shelly.call("Shelly.GetStatus", {}, function (status) {
         status.sys.installed_version = installed_version;
         status.sys.model = model;
```

Comment:
Shouldn't this line be removed as well?
running script now will have error/warnings:
`at script_2.js:14`
`RPC cb error: [model] is not defined `

Reply:
You’re right, thanks.